### PR TITLE
WV-2199: Create e2e testing for right-click menu

### PR DESCRIPTION
**Pull request by jwhurley1**
## Description

This PR adds e2e testing for the right click menu. Every menu item has a test written for it except for the following:
- Copy to clipboard: Wasn't sure how to check the clipboard.  In retrospect, maybe we can add a npm package just for this?
- Testing for Firefox.  Was not able to get Nightwatch to execute clicks in Firefox.

## How To Test
- Run
`npx nightwatch -e chromeHeadless e2e/features/ui/context-menu-test.js` to run only in the CLI or 
`npx nightwatch -e chrome e2e/features/ui/context-menu-test.js` if you want to watch the browser automate live.

### CODE REVIEW

File: e2e/features/ui/context-menu-test.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    c.expect.element(contextMenu).to.be.present;
+  },
+
+  'Adding a marker after clicking "Add Place Marker" in context menu': (c) => {
+    if (c.options.desiredCapabilities.browserName === 'firefox') {
+      return;
+    }
+    c.moveToElement(contextMenuAddMarker, 10, 10);
+    c.mouseButtonClick('left');
+    c.waitForElementVisible('#marker-pin');
```
Comment (jasontk19) [inline e2e/features/ui/context-menu-test.js:53]: I think this _should_ fail if the marker doesn't show but, wouldn't be a bad idea to add some `assert` or `expect` statement here.

**Comment by jasontk19** [comment]
For testing copy to clipboard, could you just verify that the "Copied to clipboard!" tooltip appears?  That at least confirms that some action was taken when clicking that item in the menu.

**Comment by minniewong** [comment]
@jwhurley1 Looks like one of the checks is still not working. Could you look into it?

**Comment by jwhurley1** [comment]
> @jwhurley1 Looks like one of the checks is still not working. Could you look into it? 

Yep I'm aware. I'll pick it up tomorrow after the my training course. Thanks!

**Comment by jwhurley1** [comment]
So the checks for other tests seem to be inconsistent. They all pass on my local machine. Sometimes they pass, sometimes they fail in the CI/CD pipeline. This Firefox-Windows failure had never happened before, for example. Nothing I'm working on is related to this test that failed so it's like hit or miss. Not sure what's going on.

I'll keep checking it out but it's like playing whack-a mole....

**Comment by jasontk19** [comment]
Looks like the last run of tests pass but now the branch is out of date with the release branch 😬

**Review by jasontk19 — APPROVED** [review]
:shipit:

**Pull request merged**